### PR TITLE
[Documentation] Behaviour on internally-raised events described

### PR DESCRIPTION
**Pull request by RainerKlute**
[Documentation] Differentiate Standard and Professional Editions Comp-Documentation

### CODE REVIEW

**Comment by RainerKlute** [comment]
Fixes #1565.

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile [hunk 1]
```diff
@@ -878,14 +878,23 @@ p=. Reaction effect syntax
...
+
 h4(#raising-and-processing-an-event). Raising and processing an event
 
-For the syntax of event raising, see "Raising an event":#raising-an-event.
-For the syntax of event declaration, see "Events":#events.
+For the syntax of event raising, see section "&quot;Raising an event&quot;":#raising-an-event.
+For the syntax of event declaration, see section "&quot;Events&quot;":#events.
+
+When a reaction raises an event, the subsequent behaviour of the state machine is highly dependent on the execution scheme, selected by either the "@CycleBased":#statechart_language_cyclebased (default) or the "@EventDriven":#statechart_language_eventdriven annotation. In short:
+* In the _cycle-based execution scheme_, the raised event will be added to the events that are processed by the current run-to-completion step. However, it will only be visible "downstream" in the run cycle, i.e. active states that have not yet been executed can take the event into account. When the run cycle is completed, the event will cease to exist. It can thus not be processed in any subsequent run-to-completion step.
```
Comment (BeckmaR) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: "cease to exist" is a bit drastic, maybe "stop to exist", or "not exist anymore"?

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile [hunk 2]
```diff
@@ -916,7 +925,16 @@ p(#fig_region-priorities).
 
 p=. Region priorities
 
-Not being able to raise an event and process it in "earlier" regions is a restriction you can overcome using variables. The idea is to set a "communication variable" to a certain value in a region that is processed "later". During the next RTC the earlier region could act on the variables value.
+Not being able to raise an event and process it in "earlier" regions is a restriction you can overcome using variables. The idea is to set a "communication variable" to a certain value in a region that is processed "later". During the next RTC the earlier region could act on the variables value. Another option – and in many cases the better on, is to change to the event-driven execution scheme.
```
Comment (BeckmaR) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: I have the feeling I've read this section three times now. Wouldn't it be more useful to use links and only explain it only once? At the very least, all repetitions have to be maintained.

File: plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile (hunk 1 above)
Comment (RainerKlute) [inline plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile]: This is a quite usual term. If you don't agree, you are welcome to attribute it to poetic license.

**Pull request merged**